ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -771,50 +774,37 @@ export function D3Blueprint(container, options) {
         return node;
     }
 
+    /**
+     * Draws new relationships as arcs and labels. Updates existing ones.
+     */
     function drawRelationships() {
+
         showRelationships();
 
-        let relationData = _relationGroup.selectAll('.relation')
-            .data(_d3DataHolder.visible.relationships, (d)=>(d.source._id + '_related_to_' + d.target._id));
+        const getPathId = (d) => (d.source._id + '-' + d.target._id);
```

Comment:
Both arrow funcs _might_ be more readable like
```
const getPathId = ({ source, target }) => `${source._id}-${target._id}`;
```

Reply:
Applied in 51765b60f9a72c4baf0db8bf57a3fd877f217300.